src/components/pages/production/ProductionBrief.vue

```
@@ -0,0 +1,106 @@
+<template>
+    <div>
+      <div v-if="!isEditing" class="box" @dblclick="openEditing">
+        <div v-if="isEmpty(value.description)">
+          <p>{{$t('brief.empty')}}</p>
+        </div>
+        <div class="content" v-else v-html="compileMarkdown(value.description)">
+        </div>
+      </div>
+      <div v-else class="box has-text-right">
+        <TextareaField
+          v-model="brief"
+          inputClass="textarea"
+          editable
+          :value="value.description"
+        />
+        <ButtonSimple
+          class="is-primary"
+          icon="save"
+          :class="{'is-loading': isLoading}"
+          :disabled="isLoading"
+          :text="$t('brief.save.button')"
+          @click="editBrief"
+        >
+        </ButtonSimple>
+      </div>
+    </div>
+</template>
+
+<script>
+import marked from 'marked'
+import { mapGetters, mapActions } from 'vuex'
+
+import TextareaField from '@/components/widgets/TextareaField'
+import ButtonSimple from '@/components/widgets/ButtonSimple'
+
+export default {
+  name: 'production-brief',
+  components: {
+    ButtonSimple,
+    TextareaField
+  },
+  props: {
+    value: {
+      type: Object,
+      required: true
+    }
+  },
+  data () {
+    return {
+      brief: this.value.description,
+      isEditing: false,
+      isLoading: false
+    }
+  },
+  computed: {
+    ...mapGetters([
+      'currentProduction'
+    ])
+  },
+  methods: {
+    ...mapActions([
+      'editProduction',
+      'setProduction'
+    ]),
+
+    isEmpty (str) {
+      return (!str || str.length === 0)
+    },
+
+    openEditing () {
+      this.isEditing = true
+    },
+
+    async editBrief () {
+      this.isLoading = true
+      await this.editProduction(
```

Errors are not handled here.